packages/ui/icons/OutlinedArrowDown.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="8" viewBox="0 0 14 8" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
🤔 It looks like it was wrongly exported. For correctly exported icon components there should be a viewBox of "0 0 14 8"

Reply:
Yes, it was caused by wrong export